src/index.ts

```diff
@@ -53,6 +55,7 @@ export function load(app: Application) {
     app.converter.addUnknownSymbolResolver((declaration) => {
         if (
             declaration.moduleSource === "typescript" ||
+            declaration.moduleSource === "@webgpu/types" ||
```

Comment:
I know this is needed for webgpu specific declarations, is there a better way to open this up to arbitrary `moduleSources` that you would be comfortable with? There may be other APIs that hit this blockage.

Reply:
Hmm, that's unfortunate. I'm not opposed to making an option for it, which encourages people to make an issue if they have to add a package to the list.

Reply:
Great idea, implemented and added tests